responsys_client/client.py

```diff
@@ -335,3 +335,11 @@ def _update_access_items(self, response):
         self.refresh_timestamp = utc_timestamp
         self.auth_token = parsed_response['authToken']
         self.issued_url = parsed_response['endPoint']
+
+    def unsubscribe_list_member(self, profile_list, customer_id):
```

Comment:
@lkiss80 Can you move this method up above the private methods so that all the public methods are grouped together?